packages/api/portfolioDrafts/getPortfolioDraft.test.ts

```diff
@@ -0,0 +1,52 @@
+import { DynamoDBDocumentClient, GetCommand, GetCommandOutput } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { mockClient } from "aws-sdk-client-mock";
+import { handler, NO_PORTFOLIO_PATH_PARAM, NO_SUCH_PORTFOLIO } from "./getPortfolioDraft";
+import { DATABASE_ERROR } from "../utils/errors";
+import { ErrorStatusCode, SuccessStatusCode } from "../utils/response";
+
+const ddbMock = mockClient(DynamoDBDocumentClient);
+beforeEach(() => {
+  ddbMock.reset();
+});
+
+const validRequest: APIGatewayProxyEvent = {
+  pathParameters: { portfolioDraftId: "1234" },
+} as any;
+
+describe("Validation tests", function () {
```

Comment:
```suggestion
describe("Validation tests", () => {
```

We keep being inconsistent in our test code because we'll do `async () => { }` for the `it`s but `function () { }` for the `describe`s. But we can just do `() => { }` for the `describes` too. The `describe` can't be `async` but it'd be nice to use the same syntax.